react-app/src/components/AppSideBar/AppSideBar.tsx

```diff
@@ -87,150 +89,161 @@ const AppSideBar: React.FC<AppSideBarProps> = (props) => {
                   onClickReward={transaction.openCollectRewardsModal}
                   onClickReinvest={onReinvest}
                 />
+              </div>
+            )}
+            <div
+              className={cn(
+                "hidden",
+                "sm:flex",
+                "flex-col",
+                "gap-y-4",
+                "mb-6",
+                wallet.status === ConnectionStatus.Connected
+                  ? "order-2"
+                  : "order-3"
+              )}
+            >
+              {wallet.status === ConnectionStatus.Connected && (
                 <AddressBar
                   address={userInfo?.address ?? ""}
                   onDisconnect={wallet.disconnect}
                 />
-              </div>
-            )}
+              )}
+              <Divider />
+              <AppNavigationMenu
+                activeRoute={location.pathname}
+                className={cn("pb-6")}
+                onMenuItemSelect={closeMobileMenu}
+              />
+            </div>
           </div>
+          <div
+            className={cn(
+              "flex-1",
+              "flex",
+              "relative",
+              "px-3",
+              "sm:max-w-screen-2xl",
+              "sm:px-0"
+            )}
+          >
+            <div
+              className={cn(
+                isMenuActive ? "flex" : "hidden",
+                "h-[calc(100vh_-_5.25rem)]",
+                "left-0",
+                "right-0",
+                "bg-gradient-to-b",
+                "from-white",
+                "to-likecoin-primary-bg",
+                "absolute",
+                "flex-col",
+                "gap-y-6",
+                "z-10",
+                "px-3",
+                "sm:hidden"
+              )}
+            >
+              <Divider />
+              <AppNavigationMenu
+                activeRoute={location.pathname}
+                className={cn("pb-6")}
+                onMenuItemSelect={closeMobileMenu}
+              />
+              {wallet.status === ConnectionStatus.Connected && (
+                <div className={cn("flex", "flex-col", "gap-y-6")}>
+                  <Divider />
+                  <UserInfoPanel
+                    userInfo={userInfo}
+                    onClickSend={transaction.openSendTokenModal}
+                    onClickReceive={transaction.openReceiveTokenModal}
+                    onClickReward={transaction.openCollectRewardsModal}
+                    onClickReinvest={onReinvest}
+                  />
+                  <AddressBar
+                    address={userInfo?.address ?? ""}
+                    onDisconnect={wallet.disconnect}
+                  />
+                </div>
+              )}
+            </div>
 
-          <main className={cn("block", "w-full")}>{children}</main>
+            <main className={cn("block", "w-full")}>{children}</main>
+          </div>
         </div>
       </div>
+      <Footer />
```

Comment:
We add the footer back to the sidebar?

Reply:
A container div is added to wrap the footer and the original sidebar, so when mobile nav menu is opened, `position: fixed` can be applied to container `div`, prevent user from scrolling when mobile navbar is opened. 

Also we probably won't want to show the 404 error screen with footer.